libraries/rush-lib/src/logic/pnpm/PnpmOptionsConfiguration.ts

```diff
@@ -116,6 +126,17 @@ export class PnpmOptionsConfiguration extends PackageManagerOptionsConfiguration
    */
   public readonly pnpmStore: PnpmStoreOptions;
 
+  /**
+   * The method used to set resolution mode used by PNPM.
+   *
+   * @remarks
+   * The resolution mode used by PNPM.
+   * Available options: highest, time-based, lowest-direct
+   *
+   * PNPM documentation: https://pnpm.io/npmrc#resolution-mode
```

Comment:
* This setting determines PNPM's `resolution-mode`  option. The default value is `highest`.
 * 
 * @remarks
 * Be aware that the PNPM 8 initially defaulted to `lowest` instead of  `highest`, but this 
 * was reverted in 8.6.12 because it caused confusion for users.  Rush 5.105.0 and newer
 * avoids this problem by always defaulting to `highest` when `resolutionMode` is not 
 * explicitly set in pnpm-config.json or .npmrc.
 * 
 * PNPM documentation: https://pnpm.io/npmrc#resolution-mode